saleor/graphql/product/tests/test_product.py

```diff
@@ -950,6 +950,26 @@ def test_products_query_with_filter_stocks(
     assert {node["node"]["id"] for node in products_data} == product_ids
 
 
+def test_query_products_with_filter_ids(
+    api_client, product_list, query_products_with_filter
+):
+    # given
+    product_ids = [
+        graphene.Node.to_global_id("Product", product.id) for product in product_list
+    ][:3]
```

Comment:
Let's take only 2 ids not all of them.
```suggestion
    product_ids = [
        graphene.Node.to_global_id("Product", product.id) for product in product_list
    ][:2]
```

Reply:
Oh, I didn't notice that this fixture contains only 3 elements :D Sure :D

Reply:
Done.